e2e/app/CreateAccountPage.ts

```diff
@@ -0,0 +1,218 @@
+import {ElementHandle} from 'puppeteer';
+import Button from './aou-elements/Button';
+import CheckBox from './aou-elements/CheckBox';
+import ClrIconLink from './aou-elements/ClrIconLink';
+import Label from './aou-elements/Label';
+import RadioButton from './aou-elements/RadioButton';
+import SelectComponent from './aou-elements/SelectComponent';
+import TextArea from './aou-elements/TextArea';
+import TextBox from './aou-elements/TextBox';
+import BasePage from './page-mixin/BasePage';
+
+import {defaultFieldValues} from '../resources/data/user-registration-fields';
+
+const faker = require('faker/locale/en_US');
+
+export const INSTITUTION_AFFILIATION = {
+  EARLY_CAREER_TENURE_TRACK_RESEARCHER: 'Early career tenure-track researcher',
+  UNDERGRADUATE_STUDENT: 'Undergraduate (Bachelor level) student',
+  INDUSTRY: 'Industry',
+};
+
+export const EDUCATION_LEVEL = {
+  DOCTORATE: 'Doctorate',
+};
+
+export const FIELD_LABEL = {
+  READ_UNDERSTAND_PRIVACY_STATEMENT: 'I have read and understand the All of Us Research Program Privacy Statement',
+  READ_UNDERSTAND_TERMS_OF_USE: 'I have read and understand the All of Us Research Program Terms of Use described above',
+  INSTITUTION_NAME: 'Institution Name',
+  ARE_YOU_AFFILIATED: 'Are you affiliated with an Academic Research Institution',
+  RESEARCH_BACKGROUND: 'Please describe your research background, experience and research interests',
+  EDUCATION_LEVEL: 'Highest Level of Education Completed',
+  YEAR_OF_BIRTH: 'Year of Birth',
+};
+
+export default class CreateAccountPage extends BasePage {
+
+  public async getInvitationKeyInput(): Promise<TextBox> {
+    const textbox = new TextBox(this.puppeteerPage);
+    await textbox.withCss('#invitationKey');
+    return textbox;
+  }
+
+  public async getSubmitButton(): Promise<Button> {
+    const button = new Button(this.puppeteerPage);
+    await button.withLabel({text: 'Submit'});
+    return button;
+  }
+
+  public async getNextButton(): Promise<Button> {
+    const button = new Button(this.puppeteerPage);
+    await button.withLabel({text: 'Next'});
+    return button;
+  }
+
+  public async scrollToNthPdfPage(nth: number): Promise<ElementHandle> {
+    const selector = `.react-pdf__Document :nth-child(${nth}).react-pdf__Page.tos-pdf-page`;
```

Comment:
These kinds of selectors look really brittle. I think I'd prefer to inject a test ID into the code to make the dependency explicit.

Reply:
a test-id attribute is desirable.  https://precisionmedicineinitiative.atlassian.net/browse/RW-4532 to keep track.